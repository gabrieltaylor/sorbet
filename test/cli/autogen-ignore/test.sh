#!/bin/bash

set -euo pipefail

# We're using temp files because there was some stdout/stderr syncing problems

stderr_log="$(mktemp)"
trap 'rm -rf "$stderr_log"' EXIT

main/sorbet --silence-dev-message --stop-after=namer -p autogen test/cli/autogen-ignore/autogen-ignore.rb 2> "$stderr_log"

echo --------------------------------------------------------------------------

cat "$stderr_log"
