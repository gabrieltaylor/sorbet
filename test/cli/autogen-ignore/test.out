# ParsedFile: test/cli/autogen-ignore/autogen-ignore.rb
requires: []
## defs:
[def id=0]
 type=module
 defines_behavior=0
 is_empty=0
[def id=1]
 type=module
 defines_behavior=0
 is_empty=0
 defining_ref=[Opus]
[def id=2]
 type=module
 defines_behavior=0
 is_empty=0
 defining_ref=[Autogen]
[def id=3]
 type=module
 defines_behavior=0
 is_empty=0
 defining_ref=[Proto]
[def id=4]
 type=module
 defines_behavior=0
 is_empty=0
 defining_ref=[Com]
[def id=5]
 type=module
 defines_behavior=0
 is_empty=0
 defining_ref=[Stripe]
[def id=6]
 type=module
 defines_behavior=0
 is_empty=0
 defining_ref=[Payserver]
[def id=7]
 type=module
 defines_behavior=0
 is_empty=0
 defining_ref=[Events]
[def id=8]
 type=module
 defines_behavior=1
 is_empty=0
 defining_ref=[Privacy]
[def id=9]
 type=class
 defines_behavior=1
 is_empty=0
 defining_ref=[NewDeletionManifest]
[def id=10]
 type=module
 defines_behavior=1
 is_empty=0
 defining_ref=[NewDeletionManifest RootType1]
[def id=11]
 type=class
 defines_behavior=1
 is_empty=0
 defining_ref=[NewDeletionManifest RootType2]
[def id=12]
 type=module
 defines_behavior=1
 is_empty=0
 defining_ref=[NewDeletionManifest RootType3]
## refs:
[ref id=0]
 scope=[]
 name=[Opus]
 nesting=[]
 resolved=[Opus]
 loc=test/cli/autogen-ignore/autogen-ignore.rb:3
 is_defining_ref=1
[ref id=1]
 scope=[Opus]
 name=[Autogen]
 nesting=[[Opus]]
 resolved=[Opus Autogen]
 loc=test/cli/autogen-ignore/autogen-ignore.rb:4
 is_defining_ref=1
[ref id=2]
 scope=[Opus Autogen]
 name=[Proto]
 nesting=[[Opus Autogen] [Opus]]
 resolved=[Opus Autogen Proto]
 loc=test/cli/autogen-ignore/autogen-ignore.rb:5
 is_defining_ref=1
[ref id=3]
 scope=[Opus Autogen Proto]
 name=[Com]
 nesting=[[Opus Autogen Proto] [Opus Autogen] [Opus]]
 resolved=[Opus Autogen Proto Com]
 loc=test/cli/autogen-ignore/autogen-ignore.rb:6
 is_defining_ref=1
[ref id=4]
 scope=[Opus Autogen Proto Com]
 name=[Stripe]
 nesting=[[Opus Autogen Proto Com] [Opus Autogen Proto] [Opus Autogen] [Opus]]
 resolved=[Opus Autogen Proto Com Stripe]
 loc=test/cli/autogen-ignore/autogen-ignore.rb:7
 is_defining_ref=1
[ref id=5]
 scope=[Opus Autogen Proto Com Stripe]
 name=[Payserver]
 nesting=[[Opus Autogen Proto Com Stripe] [Opus Autogen Proto Com] [Opus Autogen Proto] [Opus Autogen] [Opus]]
 resolved=[Opus Autogen Proto Com Stripe Payserver]
 loc=test/cli/autogen-ignore/autogen-ignore.rb:8
 is_defining_ref=1
[ref id=6]
 scope=[Opus Autogen Proto Com Stripe Payserver]
 name=[Events]
 nesting=[[Opus Autogen Proto Com Stripe Payserver] [Opus Autogen Proto Com Stripe] [Opus Autogen Proto Com] [Opus Autogen Proto] [Opus Autogen] [Opus]]
 resolved=[Opus Autogen Proto Com Stripe Payserver Events]
 loc=test/cli/autogen-ignore/autogen-ignore.rb:9
 is_defining_ref=1
[ref id=7]
 scope=[Opus Autogen Proto Com Stripe Payserver Events]
 name=[Privacy]
 nesting=[[Opus Autogen Proto Com Stripe Payserver Events] [Opus Autogen Proto Com Stripe Payserver] [Opus Autogen Proto Com Stripe] [Opus Autogen Proto Com] [Opus Autogen Proto] [Opus Autogen] [Opus]]
 resolved=[Opus Autogen Proto Com Stripe Payserver Events Privacy]
 loc=test/cli/autogen-ignore/autogen-ignore.rb:10
 is_defining_ref=1
[ref id=8]
 scope=[Opus Autogen Proto Com Stripe Payserver Events Privacy]
 name=[NewDeletionManifest]
 nesting=[[Opus Autogen Proto Com Stripe Payserver Events Privacy] [Opus Autogen Proto Com Stripe Payserver Events] [Opus Autogen Proto Com Stripe Payserver] [Opus Autogen Proto Com Stripe] [Opus Autogen Proto Com] [Opus Autogen Proto] [Opus Autogen] [Opus]]
 resolved=[Opus Autogen Proto Com Stripe Payserver Events Privacy NewDeletionManifest]
 loc=test/cli/autogen-ignore/autogen-ignore.rb:11
 is_defining_ref=1
[ref id=9]
 scope=[]
 name=[Google Protobuf DescriptorPool]
 nesting=[]
 resolved=[]
 loc=test/cli/autogen-ignore/autogen-ignore.rb:11
 is_defining_ref=0
[ref id=10]
 scope=[Opus Autogen Proto Com Stripe Payserver Events Privacy]
 name=[NewDeletionManifest RootType1]
 nesting=[[Opus Autogen Proto Com Stripe Payserver Events Privacy] [Opus Autogen Proto Com Stripe Payserver Events] [Opus Autogen Proto Com Stripe Payserver] [Opus Autogen Proto Com Stripe] [Opus Autogen Proto Com] [Opus Autogen Proto] [Opus Autogen] [Opus]]
 resolved=[Opus Autogen Proto Com Stripe Payserver Events Privacy NewDeletionManifest RootType1]
 loc=test/cli/autogen-ignore/autogen-ignore.rb:12
 is_defining_ref=1
[ref id=11]
 scope=[]
 name=[Google Protobuf DescriptorPool]
 nesting=[]
 resolved=[]
 loc=test/cli/autogen-ignore/autogen-ignore.rb:12
 is_defining_ref=0
[ref id=12]
 scope=[Opus Autogen Proto Com Stripe Payserver Events Privacy]
 name=[NewDeletionManifest RootType2]
 nesting=[[Opus Autogen Proto Com Stripe Payserver Events Privacy] [Opus Autogen Proto Com Stripe Payserver Events] [Opus Autogen Proto Com Stripe Payserver] [Opus Autogen Proto Com Stripe] [Opus Autogen Proto Com] [Opus Autogen Proto] [Opus Autogen] [Opus]]
 resolved=[Opus Autogen Proto Com Stripe Payserver Events Privacy NewDeletionManifest RootType2]
 loc=test/cli/autogen-ignore/autogen-ignore.rb:14
 is_defining_ref=0
[ref id=13]
 scope=[Opus Autogen Proto Com Stripe Payserver Events Privacy]
 name=[NewDeletionManifest RootType2]
 nesting=[[Opus Autogen Proto Com Stripe Payserver Events Privacy] [Opus Autogen Proto Com Stripe Payserver Events] [Opus Autogen Proto Com Stripe Payserver] [Opus Autogen Proto Com Stripe] [Opus Autogen Proto Com] [Opus Autogen Proto] [Opus Autogen] [Opus]]
 resolved=[Opus Autogen Proto Com Stripe Payserver Events Privacy NewDeletionManifest RootType2]
 loc=test/cli/autogen-ignore/autogen-ignore.rb:14
 is_defining_ref=0
[ref id=14]
 scope=[Opus Autogen Proto Com Stripe Payserver Events Privacy]
 name=[NewDeletionManifest RootType2]
 nesting=[[Opus Autogen Proto Com Stripe Payserver Events Privacy] [Opus Autogen Proto Com Stripe Payserver Events] [Opus Autogen Proto Com Stripe Payserver] [Opus Autogen Proto Com Stripe] [Opus Autogen Proto Com] [Opus Autogen Proto] [Opus Autogen] [Opus]]
 resolved=[Opus Autogen Proto Com Stripe Payserver Events Privacy NewDeletionManifest RootType2]
 loc=test/cli/autogen-ignore/autogen-ignore.rb:14
 is_defining_ref=1
[ref id=15]
 scope=[]
 name=[Google Protobuf DescriptorPool]
 nesting=[]
 resolved=[]
 loc=test/cli/autogen-ignore/autogen-ignore.rb:14
 is_defining_ref=0
[ref id=16]
 scope=[Opus Autogen Proto Com Stripe Payserver Events Privacy]
 name=[NewDeletionManifest RootType3]
 nesting=[[Opus Autogen Proto Com Stripe Payserver Events Privacy] [Opus Autogen Proto Com Stripe Payserver Events] [Opus Autogen Proto Com Stripe Payserver] [Opus Autogen Proto Com Stripe] [Opus Autogen Proto Com] [Opus Autogen Proto] [Opus Autogen] [Opus]]
 resolved=[Opus Autogen Proto Com Stripe Payserver Events Privacy NewDeletionManifest RootType3]
 loc=test/cli/autogen-ignore/autogen-ignore.rb:15
 is_defining_ref=0
[ref id=17]
 scope=[Opus Autogen Proto Com Stripe Payserver Events Privacy]
 name=[NewDeletionManifest RootType3]
 nesting=[[Opus Autogen Proto Com Stripe Payserver Events Privacy] [Opus Autogen Proto Com Stripe Payserver Events] [Opus Autogen Proto Com Stripe Payserver] [Opus Autogen Proto Com Stripe] [Opus Autogen Proto Com] [Opus Autogen Proto] [Opus Autogen] [Opus]]
 resolved=[Opus Autogen Proto Com Stripe Payserver Events Privacy NewDeletionManifest RootType3]
 loc=test/cli/autogen-ignore/autogen-ignore.rb:15
 is_defining_ref=0
[ref id=18]
 scope=[Opus Autogen Proto Com Stripe Payserver Events Privacy]
 name=[NewDeletionManifest RootType3]
 nesting=[[Opus Autogen Proto Com Stripe Payserver Events Privacy] [Opus Autogen Proto Com Stripe Payserver Events] [Opus Autogen Proto Com Stripe Payserver] [Opus Autogen Proto Com Stripe] [Opus Autogen Proto Com] [Opus Autogen Proto] [Opus Autogen] [Opus]]
 resolved=[Opus Autogen Proto Com Stripe Payserver Events Privacy NewDeletionManifest RootType3]
 loc=test/cli/autogen-ignore/autogen-ignore.rb:15
 is_defining_ref=1
[ref id=19]
 scope=[]
 name=[Google Protobuf DescriptorPool]
 nesting=[]
 resolved=[]
 loc=test/cli/autogen-ignore/autogen-ignore.rb:15
 is_defining_ref=0
--------------------------------------------------------------------------
No errors! Great job.
